src/forms/DateTimeLocalInput.jsx

```diff
@@ -4,40 +4,36 @@ import cx from 'classnames';
 
 /**
  * @module DateTimeLocalInput
- * @description renders html5 datetime-local input
+ * @description renders html5 datetime-local input, 
+ * this component is supported on mobile browsers mostly, and should be used when supported
+ * 
 */
 class DateTimeLocalInput extends React.Component {
 
 	constructor(props) {
 		super(props);
-		this.state = {
-			value: this.props.value ? (new Date(this.props.value).toISOString()).split('.')[0] : ''
-			// example: 2017-02-18T00:00:00
-			// leaving off milliseconds
-			// datetime local wont set value with milliseconds
-		};
-		this.onChange = this.onChange.bind(this);
+		this.handleChange = this.handleChange.bind(this);
 	}
 
 	/**
-	* @function onChange
-	* @description sets state and calls a callback with the value
-	* (callback used in wrapping components like datetimepicker)
+	* @function handleChange
 	* @param e Event the change event
+	* @description called when value changes and in turn calls onChange from props 
+	* (redux-form or wrapping components like datetimepicker provides this)
 	*/
-	onChange(e) {
-		this.setState({ value: e.target.value });
-		this.props.onChangeCallback && this.props.onChangeCallback(e.target.value);
+	handleChange(e) {
```

Comment:
hrm. not quite sure what you mean with ‘how its used in the jsx’ here. I thought itd be easier to distinguish the internal `handleChange` from the `onChange` prop provided if they have different names.